docs/index.md

```diff
@@ -1,52 +1,34 @@
 # HIP documentation
 
```

Comment:
Missing metadata

    :description: HIP documentation and programming guide. 
    :keywords: HIP, Heterogeneous-computing Interface for Portability, HIP programming guide

Reply:
Added.